pkg/types/types.go

```diff
@@ -6,6 +6,7 @@ import (
 )
 
 type Plan struct {
+	Features      []string                 `yaml:"features"`
```

Comment:
So can we derive this in code rather than having the user specify it in YAML in effect doing it twice?

Reply:
Yep done this in latest commits 👍